Doc/c-api/float.rst

```diff
@@ -78,6 +78,23 @@ Floating-Point Objects
    Return the minimum normalized positive float *DBL_MIN* as C :c:expr:`double`.
 
 
+.. c:macro:: Py_MATH_E
+
+   The definition of the :data:`math.e` constant.
```

Comment:
I don't think we need numbers here.  Just the reference is enough.

But you might note c type:
```suggestion
   The definition (accurate for a :c:type:`double` type) of the :data:`math.e` constant.
```
Same for other constants.

IMO, you should also reorder existing docs: `Py_MATH_PIl` must be right behind of `Py_MATH_PI`, same for `Py_MATH_El` vs `Py_MATH_E`.